### DISCUSSION TITLE
Is there any chance to see versions above 1.89?

### DISCUSSION THREAD

**Discussion by spacedrone404** [Alex313031/codium#15] [Q&A]
Or we are already reached Windows 7 headroom?

**Comment by spacedrone404** [comment]
@Alex313031 i suppose we have no chance, don't we?

**Comment by Alex313031** [comment]
@spacedrone404 Yes we do!!

It took even more work, but I now have 1.92.2 working!

Builds coming soon.

However, there is the limitation that if a VSCode extension requires Node 20, it won't work. Most don't require this though.

**Reply by spacedrone404** [reply]
> owever, there is the limitation that if a VSCode extension requires Node 20

That's incredible! I'am using node.js 18 to develop website in Windows 7, and i know that v18 will be updated till mid 2025.

It looks like that with each version it will be more and more difficult to maintain compatibility. But anyways you already have done a great job for whole win7 people! Thank you!

**Reply by Alex313031** [reply]
@spacedrone404 Are you using my Node builds from here? > https://github.com/Alex313031/node18-win7

I have installers for Node 14, 16 and 18

https://github.com/Alex313031/node14-win7/releases
https://github.com/Alex313031/node16-win7/releases
https://github.com/Alex313031/node18-win7/releases

They are also compiler optimized with -O3 (versus Node's default of -O2), so there is a (probably not noticeable but still) performance increase too.

**Reply by spacedrone404** [reply]
> installers for Node 14, 16 and 18

Nice! Good to know, i nstalled v18 manually, with adding some tweaks to system variables and path

**Reply by Alex313031** [reply]
@spacedrone404 Ahh. I used to do that too.

Those builds have patches though, to allow proper installation without any workarounds or hacks, it just works just like on Windows 10+.
No manually copying stuff, no setting environment variables, just double click and install the way you used to be able to do.

**Reply by spacedrone404** [reply]
> No manually copying stuff

Thanks! i'll try it out in a while.
and btw do we have a chance to see 115.15 mercury build?

**Reply by Alex313031** [reply]
@spacedrone404 Yeah. Working on both regular mercury and win7 version. Tryna add some good stuff from zen
